include/votca/tools/votca_mkl_boost_ublas_matrix_prod.h

```diff
@@ -0,0 +1,210 @@
+//******************************************************************************
+//  Copyright(C) 2008-2016 Intel Corporation. All Rights Reserved.
+//  
+//  The source code, information  and  material ("Material") contained herein is
+//  owned  by Intel Corporation or its suppliers or licensors, and title to such
+//  Material remains  with Intel Corporation  or its suppliers or licensors. The
+//  Material  contains proprietary information  of  Intel or  its  suppliers and
+//  licensors. The  Material is protected by worldwide copyright laws and treaty
+//  provisions. No  part  of  the  Material  may  be  used,  copied, reproduced,
+//  modified, published, uploaded, posted, transmitted, distributed or disclosed
+//  in any way  without Intel's  prior  express written  permission. No  license

```

Comment:
Did you read this? __No  part  of  the  Material  may  be distributed without Intel's  prior  express written  permission.__ I think we cannot add this will to VOTCA like is.